application/backend/src/main/resources/apps/typerefinery/components/widgets/tickervariant/clientlibs/variants/primary/functions.js

```diff
@@ -0,0 +1,107 @@
+window.Typerefinery = window.Typerefinery || {};
+window.Typerefinery.Components = Typerefinery.Components || {};
+window.Typerefinery.Page.Theme = Typerefinery.Page.Theme || {};
+window.Typerefinery.Components.Widgets = Typerefinery.Components.Widgets || {};
+window.Typerefinery.Components.Widgets.Ticker = Typerefinery.Components.Widgets.Ticker || {};
+window.Typerefinery.Components.Widgets.Ticker.Variants = Typerefinery.Components.Widgets.Ticker.Variants || {};
+window.Typerefinery.Components.Widgets.Ticker.Variants.PrimaryTicker = Typerefinery.Components.Widgets.Ticker.Variants.PrimaryTicker || {};
+window.Typerefinery.Components.Widgets.Ticker.Instances = Typerefinery.Components.Widgets.Ticker.Instances || {};
```

Comment:
Why do we need Instances for the ticker ?

Reply:
Removed unused code